docs/content/howto/configure-the-viewer/blueprint-ui.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Blueprint UI
+order: 1
+---
+
+The Rerun Viewer is configurable directly through the UI itself.
+
+## Viewer overview
+
+TODO(#5636): Screenshot of the viewer, with the blueprint and selection panels highlighted.
+
+The left panel of the viewer is the "Blueprint Panel" this shows a visual tree view representing
+the contents of the current blueprint.
+
+The right panel of the viewer is the "Selection Panel" this panels allows you to configure
+specific blueprint properties of the currently selected element.
+
+After editing the viewer you may want to [save or share the blueprint](./rbl-files.md).
+
+## Configuring Layout and Contents
```

Comment:
This section starts to feel a bit like a reference which makes me wonder if the "How to Guides" section is the right place for this page at all. Also slightly worried this will be hard to keep up to date but perhaps it's hard to get around.

One suggestion would be to do something like list the things you can do and have a video showing a couple examples. Alternatively we could add a PR checkbox to update these videos on major UI updates or something like that.

Reply:
Yeah, I'm generally against videos for that reason... they are going to rot. Those of us on linux aren't even positioned to produce the content or keep it up-to-date since we can't do video screencaps in the same style as the mac software. I don't have a great solution.